spt/OrangeBox/modules/vphysicsDLL.cpp

```diff
@@ -55,6 +55,17 @@
 		} \
 	}
 
+int __fastcall VPhysicsDLL::HOOKED_GetShadowPosition(void* thisptr, int edx, Vector* worldPosition, QAngle* angles)
+{
+	TRACE_ENTER();
+	Vector* worldpos = worldPosition;
```

Comment:
Copying this pointer seems unnecessary, can't you just use the worldPosition pointer?